sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java

```diff
@@ -584,6 +583,16 @@ public void testTriggeredFileLoadsWithTempTables() throws Exception {
         containsInAnyOrder(Iterables.toArray(elements, TableRow.class)));
   }
 
+  @Test
+  public void testTriggeredFileLoadsWithTempTables() throws Exception {
+    testTriggeredFileLoadsWithTempTables("project-id:dataset-id.table-id");
+  }
+
+  @Test
+  public void testTriggeredFileLoadsWithTempTablesPartialTableRef() throws Exception {
```

Comment:
testTriggeredFileLoadsWithTempTablesDefaultProject ?
(what we'll be doing is picking the default project which is correct according to the spec: https://github.com/apache/beam/blob/master/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryHelpers.java#L395
)

Reply:
Yes that sounds better. :)